### PR TITLE
Create UHTenancies gateway that retrieves tenancy list details

### PR DESCRIPTION
**Pull request by StevenLeighton21**
- Create gateway to retrieve tenancy list items
- Added .env support for configuring UH connection
- Moved from models to domain objects
- Stub has had `contacts`, `tenagree`, `arag` and `araction` tables added (not full schema, just columns we retrieve data from)

### CODE REVIEW

File: .circleci/config.yml_old [hunk 1]
```diff
@@ -0,0 +1,99 @@
+version: 2
```
Comment (zuren) [inline .circleci/config.yml_old]: No thanks

File: .gitignore [hunk 2]
```diff
@@ -39,3 +39,5 @@ Thumbs.db
 
 # dotCover
 *.dotCover
+
+.env*
```
Comment (zuren) [inline .gitignore]: This `*` isn't including `.env` itself. Should probably have:

```sh
.env
.env.*
```

File: LBHTenancyAPI/Program.cs [hunk 3]
```diff
@@ -13,7 +13,7 @@ namespace LBHTenancyAPI
     public class Program
     {
         public static void Main(string[] args)
-        {
+        {
```
Comment (zuren) [inline LBHTenancyAPI/Program.cs]: Why?

File: LBHTenancyAPI/Startup.cs [hunk 4]
```diff
@@ -29,6 +30,9 @@ public void ConfigureServices(IServiceCollection services)
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
         public void Configure(IApplicationBuilder app, IHostingEnvironment env)
         {
+            string dotenv = Path.GetRelativePath(Directory.GetCurrentDirectory(), "../../../.env");
```
Comment (zuren) [inline LBHTenancyAPI/Startup.cs]: Is this a reliable way to read configuration in dev/staging/production? Have we tested it yet?

**Comment by zuren** [comment]
There are some major holes in this implementation atm, but recommend we change them in a later PR:

- No test case (or implementation for) current agreement.
- No test case (or implementation for) latest arrears action event.
- No test case (or implementation for) short address - instead showing full address.
- No guarantee that test database schema lines up with the real UH schema.

**Comment by zuren** [comment]
Also, as an optimisation, we could skip the SQL query altogether when `tenancyRefs` is an empty list.

File: LBHTenancyAPI/Startup.cs (hunk 4 above)
Comment (StevenLeighton21) [reply LBHTenancyAPI/Startup.cs]: Works now, by defaulting to the option passed in the circle cfg.yml. Need a more concrete solution for production IMO

Comment (zuren) [reply LBHTenancyAPI/Startup.cs]: Lets leave it for now, it's only really relevant in dev so we'll likely change it elsewhere.

**Pull request merged**